.github/CODEOWNERS

```diff
@@ -1,70 +0,0 @@
-# This list auto requests reviews from the specified org members
```

Comment:
Why not keep the top bit saying how it works, and add one of us to a config file or something so we still have it?